<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportGhnsDialog</class>
 <widget class="QWidget" name="ExportGhnsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>150</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="exportDirectoryLabel">
       <property name="text">
        <string>Export Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exportDirectory"/>
     </item>
     <item>
      <widget class="QToolButton" name="selectDirectoryButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open-folder">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="courseListCombo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
